quiz.js

```diff
@@ -53,8 +54,9 @@ const questions = [
   function showNextQuestion() {
     showQuestion(currentQuestion + 1);
     currentQuestion += 1;
-    if (currentQuestion === questions.length - 1) {
+    if (currentQuestion === questions.length) {
```

Comment:
Nie rozumiem tej zmiany, dlaczego tak?

Reply:
Ewelina pewnie zauważyła, że wyświetlają się wszystkie pytania oprócz ostatniego. To naprawia ten błąd ale niestety wywołuje inne. Pozostańmy więc na razie na length-1 i stworzymy na rozwiązanie tej zagadki zadanie.